custom/logistics/views.py

```diff
@@ -17,7 +17,7 @@ def dispatch(self, request, *args, **kwargs):
     @property
     def page_context(self):
         try:
-            runner = ReportRun.objects.get(domain=self.domain, complete=False)
+            runner = ReportRun.objects.filter(domain=self.domain, complete=False).latest('start_run')
```

Comment:
It happens when task is killed and finally block doesn't run. In that situation report run record has set ```has_error``` to false and also ```complete``` to false. In the next run, this record is ignored since we look for last completed record. In result of this, new record is created but it's not a problem because we still recalculate data from last successful run (by last successful I mean record with complete=True and has_error=False)